include/bluetooth/mesh/main.h

```diff
@@ -63,6 +63,33 @@ typedef enum {
 	BT_MESH_PROV_OOB_ON_DEV    = BIT(15),
 } bt_mesh_prov_oob_info_t;
 
+/** Device Capabilities. */
+struct bt_mesh_prov_capabilities {
```

Comment:
The comment above the struct name and the struct name seem to be in disagreement here. Shouldn't it (based on the comment) be `struct bt_mesh_dev_capabilities`?